test/use-checkout-pricing.test.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import { renderHook, act } from '@testing-library/react-hooks';
+import { useCheckoutPricing, RecurlyProvider, Elements } from '../lib';
+
+const subscriptionPricingReturn = {
+  plan: jest.fn(() => subscriptionPricingReturn),
+  addon: jest.fn(() => subscriptionPricingReturn),
+  tax: jest.fn(() => subscriptionPricingReturn),
+  catch: jest.fn(() => subscriptionPricingReturn),
+  done: jest.fn(),
+};
+
+const mockPricing = {
+  price: {
+    now: {
+      subtotal: '10.00',
+    },
+  },
+};
+
+let checkoutPricingReturn = {
+  subscription: jest.fn(() => checkoutPricingReturn),
+  currency: jest.fn(() => checkoutPricingReturn),
+  catch: jest.fn(() => mockPricing),
+  then: jest.fn(cb => {
+    cb();
+    return checkoutPricingReturn;
+  }),
+  finally: jest.fn(() => checkoutPricingReturn),
+};
+
+describe('useCheckoutPricing', function() {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    const { Recurly } = window.recurly;
+    window.recurly.Recurly = () => {
```

Comment:
Will this leave the `Recurly` mock in place after the test is concluded?

Reply:
Such a good catch. I honestly have no idea, I'll look into it.

Reply:
I added an `afterEach` to reset the value